docs/proposals/volumes.md

```diff
@@ -236,12 +236,29 @@ setting arbitrary supplemental groups).
 An admission controller could handle allocating groups for each pod and setting the group in the
 pod's security context.
 
+#### Supporting boutique ownership and permissions
+
+Certain applications need specific ownership and permissions on volumes and our design must allow
+for this.  For example, postgresql requires permissions 0700 on its data directory.  Users should be
+able to set their own permissions on files within volumes; the starting point that pods work with
```

Comment:
"the starting point that pods with with permissions from ..."

is awkward, rephrase?